Tests/DatabaseServiceTests/Comparators/DbUserComparer.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+using DataBaseService.Database.Models;
+
+namespace DatabaseServiceTests.Comparators
+{
+    class DbUserComparer : EqualityComparer<DbUser>
+    {
+        public override bool Equals([AllowNull] DbUser x, [AllowNull] DbUser y)
```

Comment:
Convince me why we have to rewrite `Equals `and `GetHashCode` instead of creating `bool` method and then calling `IsTrue()` in tests